src/DataStructures/VariablesHelpers.hpp

```diff
@@ -0,0 +1,41 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines helper functions for use with Variables class
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Index.hpp"
+#include "DataStructures/SliceIterator.hpp"
+#include "DataStructures/Variables.hpp"
+
+/*! \ingroup DataStructures
+ *  \brief Slices the data within `vars` to a codimension 1 slice in the
+ * `sliced_dim` dimension. Which surface to slice to is determined by `slice_point`.
+ *
+ * \return Variables class sliced to a hypersurface.
+ */
+template <std::size_t VolumeDim, typename TagsLs>
+Variables<TagsLs> data_on_slice(const Variables<TagsLs>& vars,
+                                const Index<VolumeDim>& element_extents,
+                                const size_t& sliced_dim,
```

Comment:
don't need to take fundamentals by const (this is a residual thing from BitBucket, will probably have a lot of these)